README.md

```diff
@@ -1,8 +1,40 @@
-How to install this repo:
+InfraSIM -- Legacy hardware Emulator
```

Comment:
To be accurate, should it be something similar to "Infrasim-compute: bare-metal server simulator"